tools/i18n-plugin/src/main/java/org/openhab/core/tools/i18n/plugin/XmlToTranslationsConverter.java

```diff
@@ -271,7 +271,8 @@ private TranslationsSection channelTypesSection(BundleInfo bundleInfo) {
             if (stateDescription != null) {
                 stateDescription.getOptions().stream()
                         .map(option -> entry(
-                                String.format("%s.%s.state.option.%s", keyPrefix, channelId, option.getValue()),
+                                String.format("%s.%s.state.option.%s", keyPrefix, channelId,
+                                        option.getValue().replaceAll("[ :=]", Matcher.quoteReplacement("\\ "))),
```

Comment:
I'm not sure here. Replacing them all with an escaped space makes it impossible to have `:` and `=` as values (which could e.g. be used to select the separator for key-value pairs).

Reply:
Correct it should replace it properly. I've updated this.